### PR TITLE
Fill in the /tg/ readme with our own details where possible.

### PR DESCRIPTION
**Pull request by george99g**
This probably should've been done earlier.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,9 +131,9 @@ Please see [CONTRIBUTING.md](CONTRIBUTING.md)
 
 ##LICENSE
 
-All code after commit 333c566b88108de218d882840e61928a9b759d8f on 2014/31/12 at 4:38 PM PST (https://github.com/tgstation/tgstation/commit/333c566b88108de218d882840e61928a9b759d8f) is licensed under GNU AGPL v3 (http://www.gnu.org/licenses/agpl-3.0.html).
+All code after commit 333c566b88108de218d882840e61928a9b759d8f on 2014/31/12 at 4:38 PM PST (https://github.com/yogstation13/yogstation/commit/333c566b88108de218d882840e61928a9b759d8f) is licensed under GNU AGPL v3 (http://www.gnu.org/licenses/agpl-3.0.html).
```
Comment (Jordie0608) [inline README.md]: You don't want to change this line.

File: README.md [hunk 2]
```diff
@@ -131,9 +131,9 @@ Please see [CONTRIBUTING.md](CONTRIBUTING.md)
 
 ##LICENSE
 
-All code after commit 333c566b88108de218d882840e61928a9b759d8f on 2014/31/12 at 4:38 PM PST (https://github.com/tgstation/tgstation/commit/333c566b88108de218d882840e61928a9b759d8f) is licensed under GNU AGPL v3 (http://www.gnu.org/licenses/agpl-3.0.html).
+All code after commit 333c566b88108de218d882840e61928a9b759d8f on 2014/31/12 at 4:38 PM PST (https://github.com/yogstation13/yogstation/commit/333c566b88108de218d882840e61928a9b759d8f) is licensed under GNU AGPL v3 (http://www.gnu.org/licenses/agpl-3.0.html).
 
-All code before commit 333c566b88108de218d882840e61928a9b759d8f on 2014/31/12 at 4:38 PM PST (https://github.com/tgstation/tgstation/commit/333c566b88108de218d882840e61928a9b759d8f) is licensed under GNU GPL v3 (https://www.gnu.org/licenses/gpl-3.0.html).
+All code before commit 333c566b88108de218d882840e61928a9b759d8f on 2014/31/12 at 4:38 PM PST (https://github.com/yogstation13/yogstation/commit/333c566b88108de218d882840e61928a9b759d8f) is licensed under GNU GPL v3 (https://www.gnu.org/licenses/gpl-3.0.html).
```
Comment (Jordie0608) [inline README.md]: Or this one.

File: README.md (hunk 1 above)
Comment (oisin100) [inline README.md]: Not really important.

**Comment by george99g** [comment]
@Jordie0608 Alrighty then. If you want it to point to your repo instead of ours, so be it.

**Comment by Jordie0608** [comment]
It's because that commit doesn't exist on your repo and for the sake on sanity you should leave the history of licensing alone unless you decide to change it.

**Comment by oisin100** [comment]
> It's because that commit doesn't exist on your repo and for the sake on sanity you should leave the history of licensing alone unless you decide to change it.

Eh, It does exist.

**Comment by george99g** [comment]
> It's because that commit doesn't exist on your repo and for the sake on sanity you should leave the history of licensing alone unless you decide to change it.

The same commit's on our repo too, though. I thought it would be easier to link to our own repo so you don't have to come back from an external one.

**Comment by oisin100** [comment]
Agree with you there Georgy, Oh well, Its not important.

**Comment by Jordie0608** [comment]
Welp, I must've messed up the URL when checking if the commit existed. How embarrassing, my bad.

**Pull request merged**